app/Presenters/templates/Homepage/templateHowDoI.latte

```diff
@@ -0,0 +1,160 @@
+{block content}
+    <div class="container">
+        <img src="{$basePath}/img/logo.svg" id="logo">
+        <div class="center">
+            <p class="claim">Dev-stack solution based on Nette framework, which provides you with fully ready application skeleton
+                and environment <strong>to start developing a web application.</strong>
+            </p>
+            <pre>$ composer create-project instante/skeleton</pre>
+        </div>
+    </div>
+    <div class="blueline">
+        <div class="container center">
+            <div id="searchbox">
+                <span>How do I</span>
+                <input type="text" name="search_how" placeholder="Create" class="with-autocomplete">
```

Comment:
Something went wrong with the magnifier icon: 
<img width="614" alt="snimek obrazovky 2017-03-14 v 10 00 01" src="https://cloud.githubusercontent.com/assets/7109726/23893011/26fb4fbe-089d-11e7-81c4-9c8c1ff1a2d6.png">